examples/k8s_dask_plugin/Dockerfile

```diff
@@ -1,3 +1,6 @@
+# ######################
+# NOTE: For CI/CD only #
+########################
```

Comment:
They definitely can. But since one can use `ImageSpec`, a Dockerfile isn't required. I just want to let our users know that this isn't something they need to worry about.